content/applications/productivity/knowledge/management.rst

```diff
@@ -0,0 +1,109 @@
+Articles management
+===================
+
+Managing articles effectively is key to maximizing the value of your knowledge resources, whether
+working on a research project, studying for an exam, or building a knowledge database for your
+business. Knowledge allows you to fully manage your articles, from
+:ref:`creation <management/create>` to :ref:`removal <management/delete>`, through
+:ref:`structuration <management/structure>` and :ref:`sharing with others <management/share>`.
+
+You can find most tools to manage articles by clicking the vertical ellipsis button (:guilabel:`⋮`)
+at the right side of the top bar. From there, you can move, lock, delete, or make a copy of an
+article.
+
+.. image:: management/toolbox.png
+
+.. _management/create:
+
+Create articles
+===============
+
+To create articles, click the :guilabel:`+ New` button on the right side of the top bar or the
+:guilabel:`+` button next to a category or another article.
+
+.. tip::
+   Shortcut the creation of private articles; press **alt** and **c** simultaneously on your
+   keyboard.
+
+.. _management/structure:
+
+Structure articles
+==================
+
+Hierarchy
+---------
+
+Articles are organized into a hierarchical structure wherein the article on top is a parent article,
+and those underneath are called nested articles. This structure allows the grouping of related
+articles.
+
+To establish this hierarchy, create new articles by clicking the :guilabel:`+` button next to the
+parent-to-be article, or move existing articles by either dragging and dropping them under the
+parent-to-be or by clicking the vertical ellipsis button (:guilabel:`⋮`), clicking :guilabel:`Move
+To` on the toolbox and selecting the article to use as a parent.
+
+.. note::
+   The number of nested articles is **not** limited.
+
+Categories
+----------
+
+Additionally, they are divided into four categories that can be found on the left sidebar. These
+categories give articles default access rights and permissions.
+
+- :guilabel:`Favorites`: You can set any article you can access as a favorite. To do so, click the
+  star-shaped icon on the menu at the top-right of the articles. Doing so is user-specific and has
+  no impact on other users.
+- :guilabel:`Workspace`: Articles displayed in that category are available to all internal users.
+  These users have the right to read, modify or share these articles.
+- :guilabel:`Shared`: Articles displayed in that category are those you shared with internal or
+  external users or shared with you.
+- :guilabel:`Private`: Articles displayed in that category are only available to you.
+
+.. image:: management/left-sidebar-cat.png
+
+.. _management/share:
+
+Share articles
+==============
+
+You can share articles with other users or external users. To do so, open the share menu by clicking
+:guilabel:`Share` on the menu at the top-right of articles.
+
+To share one online, activate the :guilabel:`Share to web` button. Doing so generates an URL link
```

Comment:
I think it's hard not to say "article" on most of the page (and I cannot use a synonym because it's what it's supposed to be called, unfortunately). However, I say "share articles" in the previous §, so I think using "one" in the next two § works.

Reply:
I finally changed it to articles because of the (or thanks to) structuration change